src/main/java/org/jitsi/videobridge/cc/AdaptiveTrackProjection.java

```diff
@@ -96,6 +99,10 @@
      * payload type (could be VP9, could be H264, could be VP8) so it has to be
      * created on packet arrival.
      */
+    @SuppressFBWarnings(
```

Comment:
Instead of this, I wonder if we should just get rid of the `synchronized` on `getContext` (the comments there even state it isn't necessary now).